src/modules/governance/GovernanceTopPanel.tsx

```diff
@@ -80,9 +54,35 @@ export const GovernanceTopPanel = () => {
           maxWidth: 'sm',
         }}
       >
-        <ExternalLink text="SNAPSHOTS" href="https://snapshot.org/#/aave.eth" />
-        <ExternalLink text="FORUM" href="https://governance.aave.com/" />
-        <ExternalLink text="FAQ" href="https://docs.aave.com/faq/governance" />
+        <ExternalLinkButton
+          size="small"
+          variant="surface"
+          target="_blank"
+          rel="noopener"
+          href="https://snapshot.org/#/aave.eth"
+        >
+          SNAPSHOTS
```

Comment:
Would you mind wrapping the three text fields here (SNAPSHOTS, FORUM, FAQ) in `<Trans>` tags imported from `import { Trans } from '@lingui/macro';`